homeassistant/components/fully_kiosk/diagnostics.py

```diff
@@ -0,0 +1,44 @@
+"""Provides diagnostics for Fully Kiosk Browser."""
+from __future__ import annotations
+
+from typing import Any
+
+from homeassistant.components.diagnostics.util import async_redact_data
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import device_registry as dr
+
+from .const import DOMAIN
+
+DEVICE_INFO_TO_REDACT = {
```

Comment:
Let's redact:

- `SSID` (has no use, could contain names)
- `BSSID` (can be used for determining location)